clients/ossfuzz/client_test.go

```diff
@@ -180,3 +182,175 @@ func setupServer(t *testing.T) string {
 	t.Cleanup(server.Close)
 	return server.URL
 }
+
+func TestAllClientMethods(t *testing.T) {
```

Comment:
`GetCreatedAt` and `URI` aren't included in this, so `All` methods is a little misleading.

My personal preference is to break these up into smaller tests. Maybe one test for all unsupported features, so most can stay grouped. Then for the functions that actually work, those can get their own simple test func of their own

Reply:
I will include GetCreatedAt and URI.